docs/advanced/worker-configuration-reference.md

```diff
@@ -329,6 +329,23 @@ sources:
     configOpts: [NO_HZ, X86, DMI]
 ```
 
+### sources.local
+
+### sources.local.hooksEnabled
+
+Configuration option to disable/enable hooks execution. Enabled by default.
+Hooks are DEPRECATED since v0.12.0 release and support will be removed in a
+future release. Use [feature files](./customization-guide.md#feature-files)
```

Comment:
This could include links to the relevant tickets, so that interested parties can read the discussion:
* Step 1, add option: https://github.com/kubernetes-sigs/node-feature-discovery/issues/859
* Step 2, disable by default: https://github.com/kubernetes-sigs/node-feature-discovery/issues/855
* Step 3, remove hook support: https://github.com/kubernetes-sigs/node-feature-discovery/issues/856

Reply:
done